hal/src/electron/core_hal.c

```diff
@@ -42,9 +42,9 @@
 /**
  * Start of interrupt vector table.
  */
-extern char link_interrupt_vectors_location;
-extern char link_ram_interrupt_vectors_location;
-extern char link_ram_interrupt_vectors_location_end;
+extern uintptr_t link_interrupt_vectors_location[];
```

Comment:
For reference: this is now required because of array bounds checks warnings in GCC9.